src/nano_contracts/types.ts

```diff
@@ -11,6 +11,8 @@ export enum NanoContractActionType {
   WITHDRAWAL = 'withdrawal',
 }
 
+export type NanoContractArgumentType = string | Buffer | number | boolean;
```

Comment:
This can be `null` as well.

Reply:
Added on https://github.com/HathorNetwork/hathor-wallet-lib/pull/685/commits/ae048f821e498c422ba6450dd422a10f0cd7198c